ios/FluentUI/PreviewCard/PreviewCard.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc open class MSFPreviewCard: NSObject, FluentUIWindowProvider {
+    @objc open var view: UIView {
+        return hostingController.view
+    }
+    @objc public init(theme: FluentUIStyle?) {
+        super.init()
+        previewCardView = PreviewCard()
+        hostingController = FluentUIHostingController(rootView: AnyView(previewCardView.windowProvider(self)))
+        hostingController.disableSafeAreaInsets()
+    }
+    var window: UIWindow? {
+        return self.view.window
+    }
+    private var hostingController: FluentUIHostingController!
+    private var previewCardView: PreviewCard!
+}
+
+public struct PreviewCard: View {
+
+    @Environment(\.theme) var theme: FluentUIStyle
+    @Environment(\.windowProvider) var windowProvider: FluentUIWindowProvider?
+    private struct Constants {
+        static let boxMinWidth: CGFloat = 343
```

Comment:
are you adding todos to actually update with real sizes? i am kind of surprised this width is not even numbers.